src/ncp/ncp_base.cpp

```diff
@@ -1416,7 +1436,7 @@ ThreadError NcpBase::SendPropertyUpdate(uint8_t header, uint8_t command, spinel_
 
     va_start(args, pack_format);
     SuccessOrExit(errorCode = OutboundFrameBegin());
-    SuccessOrExit(errorCode = OutboundFrameFeedPacked("Cii", header, command, key));
+    SuccessOrExit(errorCode = OutboundFrameFeedPacked(SPINEL_DATATYPE_COMMAND_PROP_S, header, command, key));
```

Comment:
It bothers me a little that we appear to have two names for the same thing in spinel property == key.  Should we not reduce this to one name in the code?  I think property/prop, replacing key would improve readability.